### ISSUE TITLE
[BUG] - Positions are None

### ISSUE THREAD

**Issue by TS-Yaloshi** [lucasmonstrox/PyTibia#83]
**Describe the bug**
![image](https://github.com/lucasmonstrox/PyTibia/assets/49572893/eb9fb6aa-5ea0-4aa8-8180-326347e8e11c)

The positions are None, which results in a TypeError None

**To Reproduce**
Select Tibia window in the app. Click on Play.

**Expected behavior**
It detects the tabs correctly.

**Comment by kaybenot** [comment]
This issue also happens to me

**Comment by Gopal-Kshetri** [comment]
yes, happened similar on mine too.
![image](https://github.com/lucasmonstrox/PyTibia/assets/60227470/b63f543b-c7b2-483a-83e2-999917b90607)

**Comment by lucasmonstrox** [comment]
@TS-Yaloshi can you give me a screenshot?

**Comment by TS-Yaloshi** [comment]
> 
A screenshot of what exactly? I posted one of the program in debug mode. The error is similar to Gopal-Kshetri's screenshot.

**Comment by Gopal-Kshetri** [comment]
From what I have deduced so far is, battleye causes the screenshots of the game to be black after 2 minutes or so of logging in. Due to this we get the above error of return None. The screenshot taken are black.

Have you considered this issue and tackled in your code? @lucasmonstrox

**Comment by thiagoretes** [comment]
If the problem is related to BattleEye you should create a projection of the window using OBS Studio. Although I'm not sure if the latest version of OBS will work. Probably version 24~26 are better.

**Comment by Gopal-Kshetri** [comment]
yeah, I tried that but the pytibia project doesn't detect anything other than the game window hence unable to give the obs projection as an input to the project.

**Comment by thiagoretes** [comment]
Well, after looking into the code it needs some refactoring to be able to grab screen from OBS and inject input into Tibia window. Currently Lucas is doing all the tasks in his own OT, that's why it's not ready for use with OBS.
